spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -476,6 +503,19 @@ class PodsProjectGenerator
                 ]
               end
 
+              it 'verifies disabling prefix header generation' do
```

Comment:
nit: same test name as the one above, is this a pattern across this file?

Reply:
yes, there's the same test name for within and without a scoped target